packages/paste-core/components/slider/src/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import type {UIStates, BackgroundColorTokens} from './types';
+
+export const getColorToken = (states: UIStates): BackgroundColorTokens => {
+  if (states.disabled) {
+    return 'colorBackgroundStrong';
+  } else if (states.error) {
+    return 'colorBackgroundError';
+  } else if (states.dragging) {
+    return 'colorBackgroundPrimary';
+  } else if (states.focused) {
+    return 'colorBackgroundPrimaryStronger';
+  } else if (states.hovered) {
+    return 'colorBackgroundPrimary';
+  }
```

Comment:
suggestion: would these nested else if statements all be better off as a sort of map? `sliderColorToken[props.states]` kinda?

Reply:
Can't do. The order matters, and they're props being set to true/false and not string keys I can lookup as easily.